nunit.tests.core.csharp/TestClass.cs

```diff
@@ -0,0 +1,20 @@
+﻿// NUnit 3 tests
+// See documentation : https://github.com/nunit/docs/wiki/NUnit-Documentation
```

Comment:
There shouldn't be a space before a colon. Since the word documentation is dominant in the link, maybe we could say less?

```suggestion
// See https://github.com/nunit/docs/wiki/NUnit-Documentation
```